grabber/main.go

```diff
@@ -135,25 +135,25 @@ func listener(url string, importer *backend.Backend) {
 	}
 }
 
-func getFirstBlockNumber(importer *backend.Backend) int64 {
-	number, err := importer.GetFirstBlockNumber()
+func getLatestBlockNumber(importer *backend.Backend) int64 {
+	number, err := importer.GetLatestBlockNumber()
 	if err != nil {
-		log.Fatal().Err(err).Msg("getFirstBlockNumber")
+		log.Fatal().Err(err).Msg("getLatestBlockNumber")
 	}
 	return number
 }
-func backfill(url string, importer *backend.Backend, startFrom int64) {
-	blockNumber := getFirstBlockNumber(importer)
-	if startFrom > 0 {
-		blockNumber = startFrom
+func backfill(importer *backend.Backend, startFrom int64) {
+	blockNumber := startFrom
+	if startFrom <= 0 {
+		blockNumber = getLatestBlockNumber(importer)
```

Comment:
Technically a behavior change, but just flipped to only do rpc call if required.